tests/test_protocol_optimization.py

```diff
@@ -113,8 +115,9 @@ def poorly_spaced_path(lam):
 def simulate_protocol(lambdas_k, n_samples_per_window=100, seed=None):
     """Generate samples from each lambda window, plug into MBAR"""
     O_k, K_k = poorly_spaced_path(lambdas_k)
-    testsystem = HarmonicOscillatorsTestCase(O_k, K_k)
-    N_k = [n_samples_per_window] * len(O_k)
-    xs, u_kn, N_k, s_n = testsystem.sample(N_k, seed=seed)
+    with patch("numpy.int", int):
```

Comment:
Not for this PR: We should revisit upgrading pymbar, do we know if the previous issue we had with the newest versions were fixed?

Reply:
I'm not sure